src/components/UserSelector.tsx

```diff
@@ -74,10 +71,10 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                setAuthor(user);
```

Comment:
```suggestion
              onClick={() => setAuthor(user)}
```

you can shorten syntax here